ios-target-app/Extensions/LabelExtension.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  LabelExtension.swift
+//  ios-target-app
+//
+//  Created by Raul Piñeres Carrera on 9/09/22.
+//
+
+import Foundation
+import UIKit
+
+extension UILabel {
+    
+    convenience init(style: LabelStyle) {
+        self.init()
+        
+        translatesAutoresizingMaskIntoConstraints = false
+        
+        switch style {
+        case .big(let text, let textColor, let backgroundColor, let numberOfLines, let textAlignment, let fontSize):
```

Comment:
same here, pls see my comment about `ButtonStyle`. The same approach is applicable for `LabelStyle`

Reply:
same here